community-operators/percona/0.3.0-rc1/percona-xtradb-cluster-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,223 @@
+                "spec": {
+                    "secretsName": "my-cluster-secrets",
+                    "pxc": {
+                        "size": 3,
+                        "allowUnsafeConfigurations": false,
+                        "image": "perconalab/pxc-openshift:0.2.0",
+                        "resources": {
+                            "requests": {
+                                "memory": "1G",
+                                "cpu": "600m"
+                            }
+                        },
+                        "affinity": {
+                            "antiAffinityTopologyKey": "kubernetes.io/hostname"
+                        },
+                        "volumeSpec": {
+                            "accessModes": [
+                                "ReadWriteOnce"
+                            ],
+                            "size": "6Gi"
+                        }
+                    },
+                    "proxysql": {
+                        "enabled": true,
+                        "size": 1,
+                        "image": "perconalab/proxysql-openshift:0.2.0",
+                        "resources": {
+                            "requests": {
+                                "memory": "1G",
+                                "cpu": "600m"
+                            }
+                        },
+                        "affinity": {
+                            "antiAffinityTopologyKey": "kubernetes.io/hostname"
+                        },
+                        "volumeSpec": {
+                            "accessModes": [
+                                "ReadWriteOnce"
+                            ],
+                            "size": "2Gi"
+                        }
+                    },
+                    "pmm": {
+                        "enabled": false,
+                        "image": "perconalab/pmm-client:1.17.0",
+                        "serverHost": "monitoring-service",
+                        "serverUser": "pmm"
+                    },
+                    "backup": {
+                        "image": "perconalab/backupjob-openshift:0.2.0",
+                        "schedule": [
+                            {
+                                "name": "sat-night-backup",
+                                "schedule": "0 0 * * 6",
+                                "keep": 3,
+                                "volume": {
+                                    "accessModes": [
+                                        "ReadWriteOnce"
+                                    ],
+                                    "size": "6Gi"
+                                }
+                            }
+                        ]
+                    }
+                }
+            }
+        ]
+    categories: "Database"
+    certified: "False"
+    description: Percona Xtradb Cluster Operator is a database clustering system for horizontal scaling of MySQL.
+    containerImage: 
+    createdAt: 
```

Comment:
This should be set to the date and time of the operator container image was created